sites/fast-website/src/app/data/framework.data.ts

```diff
@@ -7,37 +7,35 @@ export interface FrameworkContentPlacementData {
 export const frameworkContentPlacementData: FrameworkContentPlacementData[] = [
     {
         body:
-            "Angular's default binding syntax will always set properties on an element. This works well for rich data, like objects and arrays, and also works well for primitive values so long as the Custom Element author has mapped any exposed attributes to corresponding properties.",
-        header: "Angular",
-        headerSubscript: "8.2.14",
+            "FAST works flawlessly with both Aurelia 1 and Aurelia 2, with full integration into the binding engine and component model.",
+        header: "Aurelia",
+        headerSubscript: "1 & 2",
```

Comment:
Should this be encoded as `&amp;` here just in case?

Reply:
We can, though it'll be a bit more involved of a change since these are just passing strings or undefined right now. I'm happy to make the change but would prefer to do it in a follow-up as it'll require changes to more than just the content.